dotcom-rendering/index.d.ts

```diff
@@ -688,6 +688,11 @@ interface CommercialConfigType {
 	ampIframeUrl: string;
 }
 
+type ServerSideTests = {
+    [k: `${string}Variant`]: "variant";
+    [k: `${string}Control`]: "control";
```

Comment:
Where does `string` come from here?

Reply:
`${string}Variant` is “any string ending in `Variant`”. As a RegEx is would be `/.*Variant/`.

https://github.com/guardian/frontend/blob/0657f594e2fd51125d28df9048c48d6eede26bb6/common/app/experiments/ExperimentsDefinition.scala#L25

And then it’s camelCased a few lines below.

Reply:
Gotcha. Thanks for the detailed explanation!